README.md

```diff
@@ -1,4 +1,127 @@
-# drafter [![Circle CI](https://circleci.com/gh/apiaryio/drafter/tree/master.svg?style=shield)](https://circleci.com/gh/apiaryio/drafter/tree/master)
+![logo](https://raw.github.com/apiaryio/api-blueprint/master/assets/logo_apiblueprint.png)
+
+# Drafter [![Circle CI](https://circleci.com/gh/apiaryio/drafter/tree/master.svg?style=shield)](https://circleci.com/gh/apiaryio/drafter/tree/master)
 
 Snowcrash parser harness
 
+### API Blueprint Parser
+Drafter is complex builder of [API Blueprint](http://apiblueprint.org). Internaly it uses [Snowcrash library](https://github.com/apiaryio/snowcrash), reference [API Blueprint](http://apiblueprint.org) parser.
+
+API Blueprint is Web API documentation language. You can find API Blueprint documentation on the [API Blueprint site](http://apiblueprint.org).
+
+Additionaly Drafter provide set of Wrappers for serialization, of parsing result, via  [SOS library](https://github.com/apiaryio/sos) into JSON and YAML format.
+
+## Status
+- [Format 1A7](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A7) fully implemented 
+
+## Install
+OS X using Homebrew:
+
+```sh
+$ brew install --HEAD \
+  https://raw.github.com/apiaryio/drafter/master/tools/homebrew/drafter.rb
+```
+
+Other systems refer to [build notes](#build).
+
+## Use
+
+### C++ library
+```c++
+#include "drafter.h"      // Blueprint Parser
+#include "SerializeAST.h" // Blueprint Result Wrapper for serialization
+#include "sosJSON.h"      // Serializer
+
+mdp::ByteBuffer blueprint = R"(
+# My API
+## GET /message
++ Response 200 (text/plain)
+
+        Hello World!
+)";
+
+// Blueprint parsing
+snowcrash::ParseResult<snowcrash::Blueprint> ast;
+drafter::ParseBlueprint(blueprint, 0, ast);
+
+std::cout << "API Name: " << ast.node.name << std::endl;
+
+// Serialization to JSON format
+sos::SerializeJSON serializer;
```

Comment:
That is not the main aim of this documentation. If we introduce serialisation here, we need to introduce sos and other stuff too. And also, not every C++ library that uses drafter will use serialisation.

I think we only want to show the `ParseBlueprint` here and leave it at that since drafter is *the* parser of API Blueprint.